content/zpages.md

```diff
@@ -45,7 +45,7 @@ from the libraries:
 
 ## /rpcz
 
-Rpcz are available at [/rpcz](http://127.0.0.1:7777/debug/rpcz). This page serves
+RPCs are available at [/rpcz](http://127.0.0.1:7777/debug/rpcz). This page serves
```

Comment:
I don't think we should change this. "Rpcz" means the Rpcz page.

Reply:
Ok, not knowing very well this software yet, I did not understand that.

So should it be "Rpcz page is available at..."?

Reply:
Yeah "Rpcz page is available at..." sounds better.